skimage/transform/seam_carving.py

```diff
@@ -47,18 +47,18 @@ def seam_carve(img, energy_map, mode, num, border=1, force_copy=True):
            http://www.cs.jhu.edu/~misha/ReadingSeminar/Papers/Avidan07.pdf
     """
 
-    utils.assert_nD(img, (2, 3))
-    image = util.img_as_float(img, force_copy)
+    utils.assert_nD(image, (2, 3))
+    img = util.img_as_float(image, force_copy)
```

Comment:
@sciunto do you want to use this PR to deprecate `img_as_*` in favour of `image_as_*`? Your call if you want to leave for later PR. But you should definitely revert this `img` back to `image`. Dunno what happened there. =)